package.json

```diff
@@ -1285,6 +1285,38 @@
             "description": "Show regular expressions in completion list.",
             "order": 20
           },
+          "dotnet.backgroundAnalysis.analyzerDiagnosticsScope": {
+            "type": "string",
+            "enum": [
+              "openFiles",
+              "fullSolution",
+              "none"
+            ],
+            "default": "openFiles",
+            "enumDescriptions": [
+              "Open documents",
+              "Entire solution",
+              "None"
```

Comment:
would have loved the descriptions and the enum member names to match :)

Reply:
These match the enum field names and user facing descriptions in VS. I think it would be reasonable to change the enum field names in Roslyn, and once the new values flow into this repo, we can rename the field names here as well. However, these are just internal implementation details, so not sure if it critical to make these changes.